src/screens/testScreen/MockExposureNotification.ts

```diff
@@ -33,9 +33,9 @@ export class MockExposureNotification implements ExposureNotification {
 
   provideDiagnosisKeys = async () => undefined;
 
-  getExposureWindows = () => [];
+  getExposureWindows = async () => [];
 
   getExposureWindowsAndroid = async () => [];
 
-  getExposureWindowsIos = async () => [];
+  detectExposureV2 = async () => [];
```

Comment:
This comment is for the wrong file @smcmurtry but on the call we talked about the naming of this function